index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Document</title>
+</head>
+<body>
+  <h1 id="title">survey Form</h1>
+  <p id="description">Lorem ipsum dolor sit amet consectetur adipisicing elit. Similique, explicabo possimus! Et impedit quam commodi nisi, qui eum molestiae aliquam eveniet dolore minima soluta dignissimos blanditiis ab eos, at harum!</p>
+  <form id="survey-form">
+    <label id="name-label" for="name">Name</label>
+    <input type="text" id="name" placeholder="Enter your name" required>
+    <label id="email-label" for="email">Email</label>
+    <input type="email" pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,4}$" id="email" placeholder="Enter your Email"required>
+    <label id="number-label" for="number">Age</label>
+    <input  type="number" min="10" max="99" pattern="[0-9]" placeholder="Enter your age"id="number">
+
+    <select id="dropdown" name="">
+      <option value="student">Student</option>
+      <option value="other">Other</option>
+    </select>
+
+    <label for="Definitely"><input type="radio" id="radio" name="user-recommend" value="definitely"> Definitely </label><br>
+    <label for="maybe"><input type="radio" id="radio" name="user-recommend" value="Maybe">Maybe</label><br>
+    <label for="NotSure"><input type="radio" id="radio" name="user-recommend" value="Not_Sure">Not Sure</label><br>
```

Comment:
Kindly you can improve the UI with a cute style 🎩👍

Reply:
- [ ] When providing feedback, it's important to remember to include an explanation, or a "why," for each change you request. By offering a reason for your suggestions, you help the recipient understand the purpose behind your request and foster a more collaborative environment. 🙃 

For example, when making a well-explained request, you could say, "Why don't you use an article tag for the .first-article element instead of a div? That would improve the HTML structure and make the page more accessible." By providing this context, you ensure that your request is clear and provide a rationale for the suggested change. 🙂 

On the other hand, if you make a request that lacks a "why," it might read as, "Why don't you use an article tag here instead of a div?" In such cases, it would be beneficial to include an explanation or reason for the suggested change. This helps the recipient better understand your perspective and increases the chances of productive collaboration.